<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetClient</class>
 <widget class="QWidget" name="WidgetClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>100</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QLabel" name="label_Ville">
       <property name="minimumSize">
        <size>
         <width>172</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string>Ville</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_CodePostal">
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string>CodePostal</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_Adresse">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string>Adresse</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_Telephone">
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string>Telephone</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_Nom">
       <property name="minimumSize">
        <size>
         <width>172</width>
         <height>34</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Liberation Sans</family>
         <pointsize>22</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_souhaits">
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>61</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>71</width>
         <height>61</height>
        </size>
       </property>
       <property name="icon">
        <iconset resource="ressources.qrc">
         <normaloff>:/app/souhait48</normaloff>:/app/souhait48</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>46</width>
         <height>46</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_biens">
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>61</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>71</width>
         <height>61</height>
        </size>
       </property>
       <property name="icon">
        <iconset resource="ressources.qrc">
         <normaloff>:/app/bien48</normaloff>:/app/bien48</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>46</width>
         <height>46</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_client">
       <property name="minimumSize">
        <size>
         <width>68</width>
         <height>61</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>71</width>
         <height>61</height>
        </size>
       </property>
       <property name="icon">
        <iconset resource="ressources.qrc">
         <normaloff>:/app/client48</normaloff>:/app/client48</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>46</width>
         <height>46</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
